include/picongpu/simulation_defines/param/laser.param

```diff
@@ -369,6 +369,108 @@ namespace picongpu
         constexpr float_X LASER_PHASE = 0.0;
     }
 
+    namespace laserExpRampWithPrepulse
+    {
+    /** Laser profile with gaussian spatial envelope and the following temporal
```

Comment:
pls intend correctly the code inside this namespace by 4 futher spaces (see existing code above)